date-carousel.js

```diff
@@ -118,14 +142,23 @@ class DateCarousel extends LitElement {
           cursor: pointer;
```

Comment:
Add .table class to css style with variables (--date-carousel-today-background, *-border-radius, *-color)